webextension/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "Cloud Storage Shield Study Web Extension Experiment (CSSSWEE)",
+  "description": "Cloud Storage Shield study as WebExtension Experiments",
+  "version": "1.0.0",
+  "applications": {
+    "gecko": {
+      "id": "cloudstorage-webextensionExperiment@shield.mozilla.org",
+      "update_url": "https://testpilot.firefox.com/files/???/updates.json"
```

Comment:
Sorry, we we can delete this line since we're not a TestPilot experiment. My bad.

Reply:
Done